<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #C7F5E8, stop: 0.1 #8DE8CE, stop: 0.49 #6DD0B3, stop: 0.5 #6DD0B3, stop: 1 #8DE8CE);
border-width: 1px;
border-color: #1D6852;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
}

QPushButton#selectButton{background-color: white}

QPushButton:hover { border-color: red }
QPushButton:pressed { border-color: rgb(0, 255, 0); }

QPainterPath{border-color: black;}
QPainterPath:hover{ border-color: red}
QPainterPath:pressed { border-color: rgb(0, 255, 0); }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get info about relations</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
